compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -530,7 +547,7 @@ class ComposeScene internal constructor(
         forEachOwner { it.clearInvalidObservations() }
     }
 
-    private var focusedOwner: SkiaBasedOwner? = null
+    internal var focusedOwner: SkiaBasedOwner? = null
```

Comment:
```suggestion
    private var focusedOwner: SkiaBasedOwner? = null
```

We can make it private after we moved hitTestInteropView